bounties_api/utils/dashboard_notification_update.py

```diff
@@ -0,0 +1,72 @@
+from notifications.models import Notification, DashboardNotification
+from bounties.utils import profile_url_for
+from std_bounties.models import Bounty, Fulfillment, Review
+
+
+def fix_it_all():
+    for notification in Notification.objects.filter(notification_name=19):
+        print()
+        print('looping with notification {}'.format(notification.__dict__))
+        print()
+        dn = DashboardNotification.objects.get(id=notification.id)
+        print()
+        print('looping with dn {}'.format(dn.__dict__))
+        print()
+        title = dn.data['bounty_title']
+        bounties = Bounty.objects.filter(title=title)
```

Comment:
@codeluggage you may not get the duplicates if you filter for bounties that are not dead. A bounty that is dead would not have associated reviews since if someone completed on it, it would transition to completed vs. dead.

Reply:
Ah, beautiful! Thank you 👍